src/Aside/MonthCalendarHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from './aside.module.css';
+
+interface CalendarHeaderProps {
+  date: Date;
+}
+
+const MonthCalendarHeader = ({ date }: CalendarHeaderProps) => {
+  return (
+    <div className={styles.calendarWidgetHeader}>
+      <span
+        id="currentDateCalendarWidget"
+        className={styles.calendarWidgetCurrentDay}
+      >
+        {date.toLocaleDateString('en-US', {
+          year: 'numeric',
+          month: 'short',
+        })}
+      </span>
+      <div className="calendar-widget-controls-group">
+        <button id="prevMonth" className={styles.switchWeeksWidget}>
+          &lt;
+        </button>
+        <button id="nextMonth" className={styles.switchWeeksWidget}>
+          &gt;
+        </button>
+      </div>
+    </div>
+  );
+};
+
+export default MonthCalendarHeader;
```

Comment:
- consistency in naming and pattern (camel vs kebab case) and wether module.css is used or not:
`<div className={styles.calendarWidgetHeader}>`
`<div className="calendar-widget-controls-group">`

- I don't think this is what id and className says it is
`       id="currentDateCalendarWidget"
        className={styles.calendarWidgetCurrentDay}`
        
- also we usually do not add attributes that are not used, in this case ids I guess are not needed if components are styled using className

- code would be cleaner by having <ArrowLeft/> and <ArrowRight />. tiny components as such can be declared in same file at the bottom